portal/templates/index.njk

```diff
@@ -19,66 +19,79 @@
+            attributes: {
+              "data-cy": "back-link"
+            }
+          }) }}
         {% endif %}
         {% block content %}{% endblock %}
       </main>
     </div>
+
     {% include "_partials/footer.njk" %}
+
     <script
-        src="/assets/js/main.js"
-        type="text/javascript"
-        integrity="sha512-HEhxQSphBlHV9OM8V0qLsCylJB4c9LbFdOC7pB9Ujg7ySL8E29ifnmtF+6i5I5ieMJk9+q+WlvhPyvyn0GxdhQ=="
-        crossorigin="anonymous"></script>
+      src="/assets/js/main.js"
+      type="module"
+      integrity="sha512-HEhxQSphBlHV9OM8V0qLsCylJB4c9LbFdOC7pB9Ujg7ySL8E29ifnmtF+6i5I5ieMJk9+q+WlvhPyvyn0GxdhQ=="
+      crossorigin="anonymous"></script>
+
     <script
-        src="/assets/js/govukFrontend.js"
-        type="text/javascript"
-        integrity="sha512-FCYqqgwMLyRhD+3K520tAkikpg9RfGv3VUQzUqo5CVwjlRxo/Ol56fxOlRhKGRxZ1LvhwYGAkyamg1Y5pX6VFg=="
-        crossorigin="anonymous"></script>
+      src="/assets/js/govukFrontend.js"
+      type="module"
+      integrity="sha512-lgazaHvhhkqqZddDJelEkMTKltMOXka6crS1q7BjVdPYHxIEZzylH47PPHYklGNX6iZFhbkp8JHwXVJp1zfQIQ=="
+      crossorigin="anonymous"></script>
+
     <script
-        src="/assets/js/mojFrontend.js"
-        type="text/javascript"
-        integrity="sha512-Ap7qengYJPIRuxlVyln9ThfoCD/SIeefliy26hN7ZG1ywrC9y2YL9o3kpKHAXT5S2gcUOuMi/GkfYLJtAaIJig=="
-        crossorigin="anonymous"></script>
+      src="/assets/js/mojFrontend.js"
+      type="module"
+      integrity="sha512-eOl0TifTsZ/1GhH6FNfs6SoJhL0shOkKCDjvLnC9+Ewwp6n6uYwIXuiQqphBMSVenttWkRWfM2Fqs6zCvneIzw=="
+      crossorigin="anonymous"></script>
+
     <script
-        src="/assets/js/maskedInputs.js"
-        type="text/javascript"
-        integrity="sha512-4XTH3IT3oQN/60lRAdBZWXMXq4/L1n5kUoGEftu+nY0Bba/7T0RDIZ62es6v+dCyNJ+uaYZTjWl3nPYv0rR6WA=="
-        crossorigin="anonymous"></script>
+      src="/assets/js/maskedInputs.js"
+      type="module"
+      integrity="sha512-4XTH3IT3oQN/60lRAdBZWXMXq4/L1n5kUoGEftu+nY0Bba/7T0RDIZ62es6v+dCyNJ+uaYZTjWl3nPYv0rR6WA=="
+      crossorigin="anonymous"></script>
+
     <script
-        src="/assets/js/mojFilters.js"
-        type="text/javascript"
-        integrity="sha512-ByfzBGRfJ1AM3hcN4bl0gILRnr3l9IDe8Um0poccVZ5qEfTpNj5r+rbYXQlEk1tL6zTdrIS2U77Kt4Jxi78Usw=="
-        crossorigin="anonymous"></script>
+      src="/assets/js/mojFilters.js"
+      type="module"
+      integrity="sha512-ByfzBGRfJ1AM3hcN4bl0gILRnr3l9IDe8Um0poccVZ5qEfTpNj5r+rbYXQlEk1tL6zTdrIS2U77Kt4Jxi78Usw=="
+      crossorigin="anonymous"></script>
+
     <script
-        src="/assets/js/disableFormSubmitOnSubmission.js"
-        type="text/javascript"
-        integrity="sha512-RwF5ml+CiOHtgzu/uJmiCoytVi+TMCZGAKWvD+meYlY8bxi+kxC2qMDDiAyi17CoSK5rzB9DO4lej7QRXz8sQQ=="
-        crossorigin="anonymous"></script>
+      src="/assets/js/disableFormSubmitOnSubmission.js"
+      type="module"
+      integrity="sha512-RwF5ml+CiOHtgzu/uJmiCoytVi+TMCZGAKWvD+meYlY8bxi+kxC2qMDDiAyi17CoSK5rzB9DO4lej7QRXz8sQQ=="
```

Comment:
it would be nice to have a tech debt PR to move these integrities to constants like EXIPs

Reply:
Agree - that'd be a massive job.